product_configurator/models/product.py

```diff
@@ -436,24 +438,36 @@ def values_available(self, attr_val_ids, sel_val_ids):
             avail = self.validate_domains_against_sels(domains, sel_val_ids)
             if avail:
                 avail_val_ids.append(attr_val_id)
-
+            elif do_raise:
+                if len(config_lines) == 1 and config_lines[0].rule_description:
```

Comment:
This has pretty limited applicability, maybe we can find a better way to raise all the errors one by one as they are fixed